docs/guides/hanadb/concepts/appbinding.md

```
@@ -0,0 +1,67 @@
+title: AppBinding CRD
+menu:
+  docs_{{ .version }}:
+    identifier: hanadb-appbinding-concepts
+    name: AppBinding
+    parent: hanadb-concepts
+    weight: 20
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)